app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -222,6 +227,9 @@ class BrowserFragment : LocaleAwareFragment(),
     }
 
     class BrowserTabsChromeListener(val fragment: BrowserFragment) : DefaultTabsChromeListener() {
```

Comment:
What's the difference if we use inner class here?

Reply:
There are not many technical difference, I just want to explicit specify what parts belong to fragment. I am OK if anyone want to change it to inner class.